packages/blockchain-extension/extension/fabric/environments/LocalEnvironment.ts

```diff
@@ -50,6 +51,8 @@ export class LocalEnvironment extends ManagedAnsibleEnvironment {
         // Delete any existing runtime directory, and then recreate it.
         await FabricEnvironmentRegistry.instance().delete(this.name, true);
 
+        LocalEnvironmentManager.instance().removeRuntime(this.name); // Seems odd that we're calling this from within the LocalEnvironment.
```

Comment:
this does seem odd,  i'm not sure why we need to delete the env entry here, also create is only called from teardown (not sure why its called from teardown) and from the localEnvironmentManager so can probably more this to the localEnvironmentManager

Reply:
I don't actually think we need this here - removed it!